tests/unit/src/test/resources/foldingRange-scala3-foldLineOnly/input/Givens.scala

```diff
@@ -0,0 +1,28 @@
+trait Foo[A]:
+  def foo: String
+
+given Foo[String] with 
+  def a: String =
+    "f" +
+      "o" +
+      "o"
+
+given Foo[String] with 
+  def a: String = 
+    "f" +
+      "o" +
+      "o"
+
+given x: AnyRef with
```

Comment:
Could you add one test with end marker?

Reply:
Added https://github.com/scalameta/metals/pull/4191/commits/81e7f54d7aa5705a66caba4a677dc6f64a41b103 :+1: